src/vs/workbench/services/views/test/browser/viewContainerModel.test.ts

```diff
@@ -63,13 +64,13 @@ suite('ViewContainerModel', () => {
 	});
 
 	test('empty model', function () {
-		container = ViewContainerRegistry.registerViewContainer({ id: 'test', title: { value: 'test', original: 'test' }, ctorDescriptor: new SyncDescriptor(<any>{}) }, ViewContainerLocation.Sidebar);
+		container = ViewContainerRegistry.registerViewContainer({ id: 'test', title: nls.localize2('test', 'test'), ctorDescriptor: new SyncDescriptor(<any>{}) }, ViewContainerLocation.Sidebar);
```

Comment:
These will be excluded from the actual localization process, right?

Reply:
Yep!